app/src/locale/translation.json

```
@@ -78,5 +78,50 @@
         "en": "Language",
         "sv-sv": "Språk",
         "ar": "اللغة"
+    },
+    "settings": {
+        "en": "Settings",
+        "sv-sv": "Inställningar",
+        "ar": "الإعدادات"
+    },
+    "about": {
+        "en": "About",
+        "sv-sv": "Om",
+        "ar": "عن"
```

```suggestion
        "ar": "معلومات"
```
Är det inte tydligare att skriva "معلومات"  "حول" istället?

---

Jo det är det